javascript/src/URDFClasses.js

```diff
@@ -228,7 +238,23 @@ class URDFJoint extends URDFBase {
 
         }
 
-        return false;
+        return didUpdate;
+
+    }
+
+}
+
+class URDFMimicJoint extends URDFJoint {
+
+    constructor(...args) {
+        super(...args);
+        this.offset = 0;
+        this.multiplier = 1;
+    }
+
+    updateFromMimickedJoint(...values) {
+        const modifiedValues = values.map(x => x * this.multiplier + this.offset);
+        return this.setJointValue(...modifiedValues);
```

Comment:
Can we override `URDFMimicJoint.setFromJointValue()` to log a warning if it's called directly? And then this function can call `super.setJointValue`. That way we can let users know if they try to call it directly and (hopefully) find out through an issue if anyone is setting mimic joints directly and see what the expected behavior should be. Something like this:

```js
console.warn( `URDFMimicJoint: Setting the joint value of mimic joint "${ this.urdfName }" will cause it to be out of sync.` );
```

Reply:
Sounds like a great idea, adding it in now.